x-pack/plugin/eql/qa/common/src/main/resources/additional_test_queries.toml

```diff
@@ -2,55 +2,64 @@
 # test_queries.toml file in order to keep the original unchanged and easier to sync with the EQL reference implementation tests.
 
 [[queries]]
+name = "betweenAdditional1"
 expected_event_ids = [95]
 query = '''
 file where between(file_path, "dev", ".json", false) == "\\TestLogs\\something"
 '''
 
 [[queries]]
+name = "betweenAdditional2"
 expected_event_ids = [95]
 query = '''
 file where between(file_path, "dev", ".json", true) == "\\TestLogs\\something"
 '''
 
 [[queries]]
+name = "cidrMatchAdditional1"
```

Comment:
why here is `additional` and next one is `stringCidrMatch` ?
Personally I would just name them `stringCidrMatchX` without the `Additional`.